try:
    print('hello')
except Exception as e:
    print(e)
    raise e